libs/api/domains/hms-loans/src/lib/models/loanOverview.model.ts

```diff
@@ -0,0 +1,142 @@
+import { Field, ObjectType } from '@nestjs/graphql'
```

Comment:
We should probably translate these models.

Reply:
I did consider translating it but I think we would need some finance specialist to do it 🙈

Reply:
I would like to push for this to be translated to english 🤮

Reply:
I created a separate ticket for translating the schema
https://www.notion.so/Translate-loan-graphql-schema-to-english-f93a385f6c6544418289d5f0ce56f051

Reply:
Api has been translated